frontend/simple/js/state.js

```diff
@@ -226,7 +313,9 @@ const actions = {
         contractId,
         type: state.contracts[contractId],
         data: state[contractId]
-      }))
+      })),
+      ephemerals: state.ephemerals,
+      users: state.users
```

Comment:
It's looking like ephemeral literalization is resulting in a huge hit to the complexity of our code. This can't be the way forward, we'll need to discuss over video chat a different way.